### ISSUE TITLE
Open access not leading to a youtube video

### ISSUE THREAD

**Issue by timokonkwo** [Open-Science-Community-Saudi-Arabia/OSCSA_Website#13]
Hi, I'm contributing to the OSCSA website as an outreachy intern and from the intern guideline, we are asked to click on any of the bullet points and it will redirect us to a youtube video,

However, I noticed that clicking the "Open Access" bullet point does not lead youtube but rather to the same page.

Thanks
![open access issue](https://user-images.githubusercontent.com/75533787/194729072-b83200ca-dbd8-4c1a-b347-fe30462e0e3e.png)

**Comment by ydv-sahitya** [comment]
I think they do no have youtube videos for that.

**Comment by BatoolMM** [comment]
Yes, there's none linked but you can link to [this Video](https://www.youtube.com/watch?v=L5rVH1KGBCY) but add where is taken from in the subpage!

Let me know if you have more questions!

**Comment by BatoolMM** [comment]
@timokonkwo do you want to work on this issue by adding a subpage for open access or do you want me to assign it to someone else?

**Comment by AminMusah** [comment]
@BatoolMM  as an outreachy intern i would like to work on this task. thank you.
